### PR TITLE
[docs] Update docs with full release of manifests, registries, and versioning

### PR DESCRIPTION
**Pull request by ras0219-msft**

### CODE REVIEW

File: docs/users/manifests.md [hunk 1]
```diff
@@ -99,11 +103,15 @@ while the remaining strings are treated as the full description.
 
 ### `"builtin-baseline"`
 
-**Experimental behind the `versions` feature flag**
+This field indicates the commit of vcpkg which provides global minimum version
+information for your manifest. It is required for top-level manifest files using
+versioning.
 
-This field indicates the commit of vcpkg which provides global minimum version information for your manifest. It is required for top-level manifest files using versioning.
+This field is a convenience field that has the same semantic as replacing your
```
Comment (ras0219-msft) [inline docs/users/manifests.md:110]:
```suggestion
This is a convenience field that has the same semantic as replacing your
```

File: docs/users/manifests.md [hunk 2]
```diff
@@ -183,19 +191,17 @@ although one can define their own.
...
+`#N` suffix to denote port-version if non-zero.
 
 See also [versioning](versioning.md#version-1) for more semantic details.
 
 ### `"overrides"`
 
-**Experimental behind the `versions` feature flag**
-
-This field enables version resolution to be ignored for certain dependencies and to use specific versions instead.
+This field pins exact versions for individual dependencies. `"overrides"` from
```
Comment (ras0219-msft) [inline docs/users/manifests.md:203]:
```suggestion
This field pins exact versions for individual dependencies.

`"overrides"` from
```

File: docs/users/versioning.md [hunk 3]
```diff
@@ -99,9 +99,14 @@ Examples:
 ## Version constraints
 
 ### `builtin-baseline`
-Accepts a Git commit ID. Vcpkg will try to find a baseline file in the given commit ID and use that to set the baseline versions (lower bounds) of all dependencies.
+Accepts a Git commit ID. Vcpkg will try to find a baseline file in the given
```
Comment (ras0219-msft) [inline docs/users/versioning.md:102]:
```suggestion
Accepts a Git commit ID. vcpkg will try to find a baseline file in the given

```

**Review by BillyONeal — APPROVED** [review]
I think we should also add examples of real world cases that work; e.g. what we told Ashon to do yesterday

File: docs/users/versioning.md [hunk 4]
```diff
@@ -161,95 +182,6 @@ For an override to take effect, the overridden package must form part of the dep
 }
 ```
 
-## Version files
```
Comment (ras0219-msft) [inline docs/users/versioning.md:164]: This section is redundant with registries

**Pull request merged**